### ISSUE TITLE
Fix tetris example

### ISSUE THREAD

**Issue by SirGnip** [pvcraven/arcade#305]
### Actual behavior:
Ran arcade/examples/tetris.py and got this exception on startup:
```
Traceback (most recent call last):
  File "tetris.py", line 274, in <module>
    main()
  File "tetris.py", line 270, in main
    arcade.run()
  File "C:\_other\repos\arcade\arcade\window_commands.py", line 214, in run
    pyglet.app.run()
  File "C:\_other\repos\venvs\arcadefull\lib\site-packages\pyglet\app\__init__.py", line 142, in run
    event_loop.run()
  File "C:\_other\repos\venvs\arcadefull\lib\site-packages\pyglet\app\base.py", line 175, in run
    self._run()
  File "C:\_other\repos\venvs\arcadefull\lib\site-packages\pyglet\app\base.py", line 187, in _run
    timeout = self.idle()
  File "C:\_other\repos\venvs\arcadefull\lib\site-packages\pyglet\app\base.py", line 314, in idle
    window.dispatch_event('on_draw')
  File "C:\_other\repos\venvs\arcadefull\lib\site-packages\pyglet\window\__init__.py", line 1313, in dispatch_event
    if EventDispatcher.dispatch_event(self, *args) != False:
  File "C:\_other\repos\venvs\arcadefull\lib\site-packages\pyglet\event.py", line 370, in dispatch_event
    if handler(*args):
  File "tetris.py", line 262, in on_draw
    self.board_sprite_list.draw()
  File "C:\_other\repos\arcade\arcade\sprite_list.py", line 532, in draw
    self.calculate_sprite_buffer()
  File "C:\_other\repos\arcade\arcade\sprite_list.py", line 350, in calculate_sprite_buffer
    name_of_texture_to_check = sprite._texture.name
AttributeError: 'NoneType' object has no attribute 'name'
```

### Expected behavior:

Tetris should be able to run against the library code on master

**Comment by DavidAntliff** [comment]
FWIW, I see the same error with `perlin_noise_2`.

**Comment by SirGnip** [comment]
I pushed up a PR that gets the sprite drawing with the perlin noise texture.  But, clicking with the mouse doesn't do anything useful.  The docstring at the top of the script states "TODO: This code doesn't work properly...".  I'm not exactly sure what clicking on a cell is suppose to do.  So, this PR is just an incremental improvement.

**Comment by pvcraven** [comment]
Closed with 2.0 release.
